soccer/src/soccer/strategy/agent/position/offense.hpp

```diff
@@ -36,6 +36,10 @@ class Offense : public Position {
 private:
     bool kicking_{true};
 
+    const float FINAL_BALL_SPEED {0.0f};
+    const float BALL_DECEL {-0.4f};
+    float dist {0.0f};
```

Comment:
i'm not convinced that `dist` needs to be a member. can we keep it local to the `state_to_task` function?